dockerfiles/Dockerfile.dev-self-contained

```diff
@@ -18,7 +18,7 @@ COPY . /watchtower
 RUN \
   cd /watchtower && \
   \
-  GO111MODULE=on CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' . && \
+  GO111MODULE=on CGO_ENABLED=0 GOOS=linux go build -a -ldflags "-extldflags '-static' -X github.com/containrrr/watchtower/cmd.version=$(git describe)" . && \
```

Comment:
```suggestion
  GO111MODULE=on CGO_ENABLED=0 GOOS=linux go build -a -ldflags "-extldflags '-static' -X github.com/containrrr/watchtower/cmd.version=$(git rev-parse --short head)" . && \
```

Reply:
git describe (`--tags` since v1.2.1 isn't annotated): `v1.2.1-2-gda56c21`
git rev-parse: `da56c21`

This shows what the "closest" release (tag) was, how many commits since that tag (2) and the ref hash.

Reply:
sure, if you prefer that, go for it, but at least add` --tags` as, as you say, not all tags are annotated. 😅

Reply:
Yep, already did.